### ISSUE TITLE
[Torrent Network] Add 2FA support

### ISSUE THREAD

**Issue by just-an-alt** [Jackett/Jackett#8846]
Torrent Network (tntracker.org) supports 2FA, however, Jackett doesn't. Seeing support for this would be great.
I do not have an invite at the moment though, sorry.

**Comment by slyre1337** [comment]
@garfield69 I can provide an inv if required, as I'm looking to get support for this as well.

**Comment by garfield69** [comment]
I was researching using 2FA with TNT, and had it enabled and was able to login with the browser.
So during the login, you get the regular username/password page, and when you click the login button, a 401 forbidden is returned, with the login page refreshed to include the token input field (along with the previously filled in username and password).
Well and good.
Then I tried to turn off 2FA on the web site, and it looks like something is broken, because now, when I use the browser to login I still get the token prompt, but using the authenticator code returns 401, so I've effectively locked myself out :-D
OH well, how sad, too bad, never mind.

**Comment by garfield69** [comment]
Does anyone have an email address for TNT support?

**Comment by ilike2burnthing** [comment]
Tried the recovery option? Even if it doesn't get you back into your account, might give you their email address.

**Comment by garfield69** [comment]
Yes, I tried the recovery option yesterday and again just now. It takes my input but gives no feedback, and nothing has come through to my registered email account, inbox or spam.
